### ISSUE TITLE
Implement subscriber model

### ISSUE THREAD

**Issue by notSanil** [PEC-CSS/Time-Table-App#12]
The app is going to work on a subscriber model, therefore there needs to be a way to enable users to subscribe to various timetables, and a way to consolidate all timetables on the home page.

- [ ] Allow users to subscribe to timetables
- [ ] Consolidate all timetables on the homepage

**Comment by rajoriaakash** [comment]
Hi, can I look over this issue?
and any tips for head starting?

**Comment by notSanil** [comment]
Yeah go ahead. You can start of by studying the database and how users are created. You'll find a subscribed databases field in each user, which is what needs to be updated upon every subscription. You can make the GUI similar to how the rest of the app looks.

**Comment by rajoriaakash** [comment]
While building the app, it shows the google-services file is missing. Should I create my own project in firebase and test it with it? Sorry, I'm kinda new to this.

**Comment by notSanil** [comment]
Yeah you can create a new firebase project for testing purposes. We are planning to keep a database schema similar to this, so you can mimic this in your own database.
![DataSchema](https://user-images.githubusercontent.com/44319106/194483047-f751c83d-f6fe-43c9-97e7-5d8ea3abc332.png)

**Comment by rajoriaakash** [comment]
Okay, so firstly i think i should take the login implementation and database integration issues also? then I could proceed with this.
Can I take them?

**Comment by notSanil** [comment]
I would suggest that you make a basic login implementation, or perhaps even a small button that adds a basic user into the database. As far as the integration goes, it would require creating CRUD ops and integrating them properly in the entire app, so it's upto you if you wanna take that as well.

**Comment by rajoriaakash** [comment]
Hi, sorry for the delay i got busy in exams and college stuff and just realised hacktoberfest is coming to an end :). Can I take the database integration issue first ?

**Comment by notSanil** [comment]
Hi! Yes sure you can take that, it's been currently assigned to someone else, but since there's been no activity, you can go ahead and do that.
